sw/device/tests/penetrationtests/README.md

```diff
@@ -2,3 +2,28 @@
 
 The penetration testing framework is based on the [Cryptotest](../crypto/cryptotest/README.md) framework.
 The purpose of this framework is to perform side-channel analysis (SCA) and fault injection (FI) attacks on the FPGA as well as on the chip.
+
+To build the penetration testing framework for the ChipWhisperer FPGA board, run:
```

Comment:
I assume these instructions are valid for both CW-305 and CW-310?

Reply:
Good point, no this instruction is just for the CW310 - I'll update the commit. Support for the CW305 is planned at a later stage.